package.json

```diff
@@ -114,13 +110,16 @@
     "debug:server": "cross-env NODE_CONFIG_ENV=development nodemon --inspect server.js 8080",
     "profile:server": "cross-env NODE_CONFIG_ENV=development node --prof server 8080",
     "benchmark:badge": "cross-env NODE_CONFIG_ENV=test node scripts/benchmark-performance.js --iterations 10100 | node scripts/capture-timings.js --warmup-iterations 100",
-    "prestart": "run-s --silent depcheck defs features",
-    "start": "concurrently --names server,frontend \"npm run start:server\" \"cd frontend && cross-env GATSBY_BASE_URL=http://localhost:8080 gatsby develop --port 3000\"",
+    "prestart": "run-s --silent depcheck defs",
+    "start": "concurrently --names server,frontend \"npm run start:server\" \"npm run docusaurus:start\"",
```

Comment:
When you run the built version of the docs (i.e: `npm run build` and serve it from scoutcamp), the website performs really well because it is just serving static HTML files.

The dev server (with hot reload when you change the files and all that jazz) is a bit of a resource hog, especially the first time you start it. It does seem to get a bit better once docusaurus has cached some stuff, but your first experience of running `npm start` is likely to be a bit sluggish. I think the fact we have over 600 pages doesn't help but in general this is an acknowledged issue: https://github.com/facebook/docusaurus/issues/4765 It is not great dev experience, but I can live with it. Its not an issue that will affect production. This was kinda true of gatsby too, but docusaurus is way worse. If it turns into a big pain point, we could consider having 2 variants of the dev server:
- A "I want to work on the website" command (that runs `docusaurus start` so you get hot reloads for the frontend) and
- A "I want to work on the badge server" command (that runs `npm run build` and serves it from scoutcamp)

For now I am going to ignore this issue.

Reply:
Just thinking this through a bit more..

If we think the performance of the webpack devserver is terrible, I think changing the default behaviour of `npm start` in dev to just compile the frontend and serve it with no watch on the frontend (but keep nodemon doing hot reloads for the backend) does not actually lose us very much. I've deleted almost all of the frontend code so there is less to watch :) I think it would be useful to get another opinion on the dev experience here.

Reply:
i don't think i'll have time today but i'll pull your branch down locally and give it a test drive soon